### ISSUE TITLE
problems and solutions 

### ISSUE THREAD

**Issue by clearlove77777777** [manishmanu/VideoVirtualTryon-HCAI-Project#2]
### 1.the project has been test on cuda 11.4(also 11.3),pytorch 1.1.2+cu113 torchaudio 0.12.0+cu113  torchversion 0.13.0+cu113,python 3.9.18 win11, my GPU is RTX3070TI and driver version is 551.23.
2.the flownet2 is extremely hard to install and this(https://blog.csdn.net/Strive_For_Future/article/details/120940839)  might help.make sure  install this version of FlowNet2.  https://github.com/andrewjong/flownet2-pytorch-1.0.1-with-CUDA-10/tree/1e365dbf7473a8ea6801a00d0f39a3081c92b347.
3.if you only have one gpu .plz set gpu_ids =0.
4.if you run this on  windows,  change the   `num_workers=int(opt.nThreads))` in test/data/custom_dataset_data_loader_test.py  as   `num_workers=0` .
5. fix some numpy problems   `edge = torch.FloatTensor((edge.detach().numpy() > 0.5).astype(np.int))` in test/test.py as   `edge = torch.FloatTensor((edge.detach().numpy() > 0.5).astype(np.int_))`
